misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/launch/ProgramLauncher.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Reply:
Only renamed the class name. We dont need to change the copyright year right?

Reply:
Then no need to change it. PR looks like you added a new class